lib/gds_api/test_helpers/content_store.rb

```diff
@@ -7,7 +7,7 @@ module TestHelpers
     module ContentStore
       include ContentItemHelpers
 
-      def content_store_endpoint(draft = false)
+      def content_store_endpoint(draft = false) # rubocop:disable Style/OptionalBooleanParameter
```

Comment:
So the linter wants this changed to `def content_store_endpoint(draft: false)`?

Reply:
Yup that's correct. On the basis that method calls like `content_store_endpoint(true)` don't really mean anything but `content_store_endpoint(draft: true)` does. 

I've fixed this as a breaking change in https://github.com/alphagov/gds-api-adapters/pull/1034/commits/4a7954c4ddcf92c481a44ab5c9f472dd55b26848 (though I don't think it's a breaking change that'll affect any apps as I can't find any apps - outside of this one - that use)